rllib/env/multi_agent_env_runner.py

```diff
@@ -350,6 +350,8 @@ def _sample_timesteps(
 
                 # Create a new episode instance.
                 self._episode = self._new_episode()
+                self._make_on_episode_callback("on_episode_created", self._episode)
```

Comment:
We could even write `self.make_on_episode_callback("on_episode_created")` as `self._episode` is used anyways.

Reply:
fixed (in next PR :) )